src/app/(fullscreen)/detail/_components/share-modal.tsx

```
@@ -82,7 +84,13 @@ export default function ShareModal({
           <ShareKakaoIcon />
         </button>
         <button onClick={handleCopyLink}>
-          <ShareLinkIcon />
+          <Image
+            src={"/images/share-link.png"}
+            alt="공유아이콘"
+            width={70}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)